cylc/flow/tests/conftest.py

```diff
@@ -30,3 +34,52 @@ def _cycling_mode(integer=True):
             (INTEGER_CYCLING_TYPE if integer else ISO8601_CYCLING_TYPE)
         )
     return _cycling_mode
+
+
+@pytest.fixture
+def mock_glbl_cfg(tmp_path, monkeypatch):
+    """A Pytest fixture for fiddling globalrc values.
+
+    * Hacks the specified `glbl_cfg` object.
+    * Can be called multuple times within a test function.
```

Comment:
Probably nitpicking but there's a typo in this docstring
```suggestion
    * Can be called multiple times within a test function.
```